igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisBuildConverter.groovy

```diff
@@ -39,6 +40,14 @@ class TravisBuildConverter {
         return genericBuild
     }
 
+    static GenericBuild genericBuild(V3Build build, String baseUrl) {
+        GenericBuild genericBuild = new GenericBuild(build.state == 'started', build.number, build.duration,TravisResultConverter.getResultFromTravisState(build.state), build.repository.slug, url(build.repository.slug, baseUrl, build.id))
```

Comment:
It's not clear what's being initialized here - could you switch to keyword params? e.g. `new GenericBuild(param1: x, param2: y)`